numhask/src/NumHask/Algebra/Abstract/Field.hs

```diff
@@ -18,14 +17,14 @@ module NumHask.Algebra.Abstract.Field
   )
 where
 
-import           NumHask.Algebra.Abstract.Group
-import           NumHask.Algebra.Abstract.Ring
-import           NumHask.Algebra.Abstract.Multiplicative
-import           NumHask.Algebra.Abstract.Additive
-import           NumHask.Data.Integral
-import           Data.Bool                      ( bool )
-import qualified Prelude                       as P
-import           Data.Complex                   ( Complex(..) )
+import Data.Bool (bool)
+import Data.Complex (Complex(..))
+import NumHask.Algebra.Abstract.Additive
+import NumHask.Algebra.Abstract.Group
+import NumHask.Algebra.Abstract.Multiplicative
+import NumHask.Algebra.Abstract.Ring
+import NumHask.Data.Integral
+import qualified Prelude as P
```

Comment:
I would perfer the style of the imports above. Brittany is capable of formatting them in this way. But this is just a thought and not really important.

Reply:
I don't like how they get jammed up on an arbitrary right column margin - we've got 80 columns and we should use them etc.  Happy to align with qualified etc, because it's always the same - so I don't have to do anything if cutting pasting (that's the major problem with the style - the cut/paste)